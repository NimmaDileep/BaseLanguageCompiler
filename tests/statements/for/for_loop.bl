for(var i: Int = 1; i <= 5; i = i + 1) {
	print(i);
}

for(var i: Int = 5; i > 0; i = i - 1) {
	print(i);
}

for(var i: Int = 1; i <= 10; i = i * 2) {
	print(i);
}

var j: Int = 10
for( ; j <= 15; j = j + 1) {
	print(j);
}

