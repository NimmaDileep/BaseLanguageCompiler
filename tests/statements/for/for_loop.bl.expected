1
2
3
4
5
5
4
3
2
1
1
2
4
8
10
11
12
13
14
15